### ISSUE TITLE
[Bug] Step "Run markdown lint" ends with en error

### ISSUE THREAD

**Issue by ricardojfc** [skills/test-with-actions#44]
### Summary

Step "Run markdown lint" ends with en error and the `README.md` isn't updated

### How to reproduce

1. Open a new browser tab, and work on the steps in your second tab while you read the instructions in this tab
2. Go to the **Actions tab**.
3. Click **New workflow**.
4. Search for "Simple workflow" and click **Configure**.
5. Name your workflow `ci.yml`.
6. Update the workflow to remove all steps other than the "checkout" step.
7. Add the following step to your workflow:
    
    ```yaml
    - name: Run markdown lint
      run: |
        npm install remark-cli remark-preset-lint-consistent
        npx remark . --use remark-preset-lint-consistent --frail
    ```
    
    > Even after the code is indented properly in `ci.yml`, you will see a build error in GitHub Actions. We'll fix this in the next step.
    
8. Click **Start commit**, and choose to make a new branch named `ci`.
9. Click **Propose a new file**.
10. Click **Create pull request**.
11. Wait about 20 seconds then refresh this page (the one you're following instructions from). [GitHub Actions](https://docs.github.com/en/actions) will automatically update to the next step.

![Image](https://github.com/skills/test-with-actions/assets/112181786/0d74da0b-1acc-499b-9d0a-4340913f5d0a)

**Comment by Gabor12321** [comment]
the task: "Update the code in the ci branch to get the test to pass." Clue: on line 18 there is a file. the issue has something to do with markdown syntax. i personally asked chat gpt, and thats how i reached to the solution after a long time.. good luck!

**Comment by javeedambu** [comment]
The following line is causing the warning and error

> npx remark . --use remark-preset-lint-consistent --frail

The "." is asking to check all files for markdown syntax and its warning that "resume.md" is not following the syntax.

You can use "_" or "*" to markdown as italics font.
However when you use "consisent" as in the code above, then you should only use either "_" or "*" throughout the file.
"resume.md" started using "_" and then later it also used "*", hence the warning.
(ref:https://www.npmjs.com/package/remark-lint-emphasis-marker/v/3.1.0)

The "--frail" will exits as failed when there is a warnings.
(ref: Search for "--frail" in the URL:https://github.com/remarkjs/remark-lint)

In short if you where to replace "*Git*" with "_Git_" in the "resume.md"  then you will not have any errors.

But then we don't get to do the LAB as per the instructions provided.

**Comment by sinsukehlab** [comment]
This error is intentional.
> Even after the code is indented properly in `ci.yml`, you will see a build error in GitHub Actions. We'll fix this in the next step.

**Comment by hectorsector** [comment]
@sinsukehlab is right, but I agree that the instructions expect the learner to do a lot of heavy lifting with reading so I'll leave this open but it's feedback about the course structure mostly.

**Comment by gradybarrett** [comment]
> @sinsukehlab is right, but I agree that the instructions expect the learner to do a lot of heavy lifting with reading so I'll leave this open but it's feedback about the course structure mostly.

> This error is intentional.

I don't think this is the issue. The main issue is that the Step 2 guide is written with a frame a reference that speaks to unit (or other tests), not linting. So the person trying to follow along is left wondering where the tests are - spoiler, there are none. I'm guessing this tutorial was originally written using tests, but then ported to linting at some point? And if so, step 2 was never updated to talk about linting instead of testing - including the screenshots, which still show a matrix run and test failures, which causes even more confusion.

**Comment by Arry-eng** [comment]
@gradybarrett is right in pointing out that for a new learner it is difficult to find and fix errors by just looking at few lines of error message. There were no instructions to run the job in "debug mode" either. 
Looking for "remake" run script errors is not the intention of this particular skills project. 
![image](https://github.com/user-attachments/assets/0dae8fa9-6167-4061-8760-7373cdb3dc15)

**Comment by MEZA87** [comment]
Sir i identify yor work flow for githib copilot account right? Do ypu need help for this problem? Sory my enhlish its not perfect.
